docs/included.md

```diff
@@ -431,6 +431,14 @@
     [Tutorial](project:./tutorials/Binary_Contingency_Scores.md)
   - 
     [Wikipedia](https://en.wikipedia.org/wiki/Precision_and_recall); [Success ratio (WWRP/WGNE Joint Working Group on Forecast Verification Research)](https://www.cawcr.gov.au/projects/verification/#SR)
+* -  
+    - Positive Predictive Value (Success Ratio)
+  -
+    [API](api.md#scores.categorical.BasicContingencyManager.positive_predictive_value)
```

Comment:
This link isn't working on branch 228. @tennlee does the positive_predictive_value function need adding to the API?

Reply:
I stand corrected - this link is now working fine in branch 228. Excellent!